Chap_API_Sync_Access.tex

```diff
@@ -165,6 +165,10 @@ \subsection{Fence-related attributes}
 available to each participant at the end of the operation. By default, this will include all job-level information that was locally generated by \ac{PMIx} servers unless excluded using the \refattr{PMIX_COLLECT_GENERATED_JOB_INFO} attribute.
 }
 %
+\declareAttributeNEW{PMIX_LOCAL_COLLECTIVE_STATUS}{"pmix.loc.col.st"}{pmix_status_t}{
+Status code for local collective operation being reported to the host by the server library
```

Comment:
I had to look back at the PR comment to understand the use case for this
 > e.g., when the server library aggregates all local participation before calling the host

Can we add such a quick note example to the definition here for clarity?

So this is a status value indicating progress in the collective or a status value indicating if the collective completed successfully/erroneously? If the former, then what status values might it send up?

I'm just trying to better understand the expected usage of this attribute.

Reply:
I addressed this - please see if the revised text helps. Short answer is that it is the latter - a single upcall reporting completion status.

Reply:
That helps. Thanks